core/src/main/scala/kafka/server/ReplicaAlterLogDirsThread.scala

```diff
@@ -97,6 +97,16 @@ class ReplicaAlterLogDirsThread(name: String,
     }
   }
 
+  // For ReplicaAlterLogDirsThread, if the partition is marked as failed due to an unknown exception and the partition fetch
+  // is suspended, the paused cleanup logic of the partition needs to be canceled, otherwise it will lead to serious unexpected
+  // disk usage growth.
```

Comment:
```suggestion
  // For ReplicaAlterLogDirsThread, if the future partition is marked as failed due to an unknown exception and the partition fetch
  // is suspended, the paused cleanup logic of the current partition needs to be resumed, otherwise it can lead to unexpected
  // disk usage growth.
```